### ISSUE TITLE
Error "The specified deps.json ... does not exist"

### ISSUE THREAD

**Issue by PureWeen** [dotnet/efcore#28335]
Moved issue from https://github.com/dotnet/maui/issues/8114

### Description

When I create a new Net Maui APP Project in the Visual Studio 2022 17.3.0 Preview 2 and install EntityFramework 6, I always get the errormessage "The specified deps.json [C:\Users\matth\Desktop\Projekte\Test\bin\Debug\net6.0-android\Test.deps.json] does not exist" after entering "Enable-Migrations"

Everything works fine in Visual Studio 2022 17.2.4 in a WPF App

Dotnet Version:6.0.400-preview.22301.10

I uploaded the project to my github https://github.com/MatthiasBae/Error-deps.json

### Steps to Reproduce

1. Create new Net Maui APP Project
2. Install EntityFramework from Nuget Paket Manager
3. Enter "Enable-Migrations" into the PM Console

### Version with bug

Unknown/Other (please specify)

### Last version that worked well

Unknown/Other

### Affected platforms

Windows

### Affected platform versions

Windows 11 Version 21H2 Build 22000.675

### Relevant log output

```shell
PM> Enable-Migrations
The specified deps.json [C:\Users\matth\Desktop\Projekte\Test\bin\Debug\net6.0-android\Test.deps.json] does not exist
```

**Comment by ajcvickers** [comment]
Duplicate of #16386.

**Comment by MAUIoxo** [comment]
Please open it again this issue is still not solved with the Latest VS 17.9 Preview 1 and also .NET 8 in combination with .NET MAUI. I'm writing it here since the .NET MAUI Bug was closed as it is not related to MAUI, but we can't use it in a MAUI project. So please open it again

**Comment by leonedev** [comment]
Same problem of @MAUIoxo, please FIX IT!

**Comment by KerryXEX** [comment]
Also occurs when trying to create Migrations on latest MAUI project templat on macOS (ARM).

**Comment by smola93** [comment]
This is another big issue on MAUI .NET 8 version, would it be reopened and fixed?

**Comment by Pakalo** [comment]
Hello
Did someone found a solution for this problem cause i have same issue with mac os and Jetbrain Rider and nobody explain in the duplicated issue how to fix this in maui ?
Thank you

**Comment by JoseMarquesJM** [comment]
Eu estou a ter o mesmo problema com o blazor e o Ef core

**Comment by Fludem** [comment]
> Also occurs when trying to create Migrations on latest MAUI project templat on macOS (ARM).

Experience the same issue

**Comment by Fludem** [comment]
@Arthri I agree. I also experienced this on MacOS (ARM). Android/iOS not targeted. Windows Target probably caused it in my situation. It was a while ago.

This problem and numerous others have made .NET MAUI unviable for any Desktop App. 
I understand this is beyond this issue, but I thought I'd mention it here.

I had several other issues with the tooling on MacOS. The problems would always be with using MAUI, targeting Windows, or both.

I spent 2-3 weeks on a project, and I can honestly say I probably spent 80% of that time-solving problems with Tooling or other issues that weren't the code I'd written.
